Tests/test_file_tiff.py

```diff
@@ -103,6 +104,10 @@ def test_int_resolution(self):
         im._setup()
         self.assertEqual(im.info['dpi'], (71., 71.))
 
+    def test_save_setting_missing_resolution(self):
+        Image.open("Tests/images/10ct_32bit_128.tiff").save(
+            BytesIO(), format="tiff", resolution=123.45)
```

Comment:
@	anntzer This test checks a file can be opened and saved without raising any exceptions (which I can confirm happens on master, see below), which is good, but is it possible to assert something as well? Perhaps open the saved file and verify some relevant values?

---

```
ERROR: TestFileTiff.test_save_setting_missing_resolution
----------------------------------------------------------------------
Traceback (most recent call last):
  File "Tests/test_file_tiff.py", line 483, in test_save_setting_missing_resolution
    BytesIO(), format="tiff", resolution=123.45)
  File "/usr/local/lib/python2.7/site-packages/PIL/Image.py", line 1685, in save
    save_handler(self, fp, filename)
  File "/usr/local/lib/python2.7/site-packages/PIL/TiffImagePlugin.py", line 1483, in _save
    offset = ifd.save(fp)
  File "/usr/local/lib/python2.7/site-packages/PIL/TiffImagePlugin.py", line 740, in save
    data = self._write_dispatch[typ](self, *values)
KeyError: None
```